common/models/role.js

```diff
@@ -218,13 +218,16 @@ module.exports = function(Role) {
    * @param {Boolean} isOwner True if the user is an owner.
    * @promise
    */
-  Role.isOwner = function isOwner(modelClass, modelId, userId, principalType, callback) {
-    if (!callback && typeof principalType === 'function') {
+  Role.isOwner = function isOwner(modelClass, modelId, userId, principalType, accessToken, callback) {
```

Comment:
You should pass full `options` object, as it may contain additional information beyond `accessToken`, for example the current transaction to use when talking to an SQL datastore.